pkg/cloudprovider/cloud.go

```diff
@@ -167,3 +167,7 @@ func (s ServiceAnnotation) GetValue(key string) (string, bool) {
 	val, ok := s[key]
 	return val, ok
 }
+
+const (
+	LBAnnotationAllowSourceRange = "service.beta.kubernetes.io/load-balancer-source-ranges"
```

Comment:
This sort of leaves NodePorts in limbo - is that a problem?  Or conversely, once we have proper network policy, does this become obsolete?